llama_index/embeddings/voyageai.py

```diff
@@ -85,3 +86,11 @@ async def _aget_text_embeddings(self, texts: List[str]) -> List[List[float]]:
         return await self._model.aget_embeddings(
             texts, model=self.model_name, input_type="document"
         )
+
+    def get_general_text_embedding(
```

Comment:
it is just getting one embedding for a single text. could you add a comment to distinguish it with get_text_embedding?
could you also add async function for it

Reply:
The docstring distinguishes it just fine I think (i.e. get an embedding with any input type)

Reply:
I can't edit the PR, but feel free to add async in another PR @thomas0809